google/resource_google_folder_organization_policy.go

```diff
@@ -38,7 +39,7 @@ func resourceGoogleFolderOrganizationPolicyCreate(d *schema.ResourceData, meta i
 
 func resourceGoogleFolderOrganizationPolicyRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
-	folder := d.Get("folder").(string)
+	folder := canonicalFolderId(d.Get("folder").(string))
```

Comment:
You should also call this method at line 73 and 91 to ensure the create, delete and update method also work.

Can you also update one of the acceptance test to use the id only in for the folder field:
```go
resource "google_folder_organization_policy" "list" {
  folder     = "replace(${google_folder.orgpolicy.name}, 'folders/','')
  ...
```

Reply:
Done.